packages/template-retail-react-app/app/pages/checkout/partials/payment.jsx

```diff
@@ -152,6 +152,12 @@ const Payment = () => {
             }
             disabled={appliedPayment == null}
             onEdit={() => goToStep(STEPS.PAYMENT)}
+            editLabel={
+                <FormattedMessage
```

Comment:
A React Component is passed, but it is used as a component, which is kinda odd. Why not use intl.formatMessage and have the ToggleCard consumes as is?

Reply:
Oh yeah. Makes sense. Replaced it with `formatMessage({...})`. Thanks!